Constructive Code Review/outline.txt

```diff
@@ -0,0 +1,205 @@
+- Splash screen
+    - PR screenshot: “Your code is bad and you are bad. Have a bad day.”
+- Importance
+    - Who's done code review before?
+        - Who's done it sync?
+            - We don't really do that, aside from maybe some pair
+              programming.
+        - Async?
+    - Pretty much all code at Mozilla gets reviewed before it gets
+      merged.
+        - It's a great way to…
+            - Improve the code
+                - I like to think I'm good at this, but people still
+                  find improvements to make in my stuff almost every
+                  time.
+            - Learn the codebase and the craft
+                - It informs other people what's going on in the
+                  codebase so you're not stuck with the sole
+                  responsibility of maintaining it.
+                - It's as much about teaching as fixing. It's a major
+                  way we bring junior people up the ladder.
+                - Tests may show the code works, but reviews ensure
+                  it's easy to understand.
```

Comment:
That's why I said "show" instead of "prove", but yes, maybe I made the dichotomy too black-and-white. I'll reexamine.